packages/config/src/balance/balance.ts

```diff
@@ -100,6 +103,29 @@ function system(): SystemBalanceConfig {
   };
 }
 
+function systemEquilibrium(id: number): EquilibriumSystemBalanceConfig {
```

Comment:
good function name pick, to emphasize that they're the only ones using it like this :)

Reply:
I don't think that anyone else will use their pallet. Their code is not open source.